src/coreclr/src/vm/codeman.cpp

```diff
@@ -6175,11 +6175,12 @@ int NativeUnwindInfoLookupTable::LookupUnwindInfoForMethod(DWORD RelativePc,
         if (RelativePc < pNextFunctionEntry->BeginAddress)
         {
             PTR_RUNTIME_FUNCTION pFunctionEntry = pRuntimeFunctionTable + i;
+            
             if (RelativePc >= pFunctionEntry->BeginAddress)
             {
                 return i;
+                break;
```

Comment:
It seems the break should stay where it was.

Reply:
Sigh. Yes.